docs/process-based-load-script.md

```diff
@@ -0,0 +1,114 @@
+# Processes based load script
+
+* [Requirements](#requirements)
+* [Parameters description](#parameters-description)
+* [Examples](#examples)
+
+## Requirements
+
+* native libs
+    * libindy >= 1.4.0
+
+* python lib
+    * python3-indy >= 1.4.0
+    * libnacl == 1.6.1
+    
+## Parameters description
+
+'-c', '--clients' : Number of independent processes to run.
+Each process will use its own pool handle and wallet and will send requested number of specified pool txns.
+If less or equal than 0 value provided the number of processes will be equal to number available CPUs.
+Default value is 0.
+
+'-g', '--genesis' : Path to file with genesis txns for the pool to connect to.
+
+'-s', '--seed' : Seed that will be used to generate submitter did.
+
+'-w', '--wallet_key' : Key to access encrypted wallet
+
+'-n', '--num' : Number or txns in one batch. If timeout param omitted only one batch will be sent.
+
+'-t', '--timeout' : Timeout between batches. If omitted only one batch will be sent.
+
+'-d', '--directory' : Directory to store output csv files:
+* args - full command line of test run
+* total - status of all sent txns with timestamps of each step
+* successful - request and response for successful txns
+* failed - request and error provided by libindy
+* nack_reject - request and error provided by pool
+
+'--sep' : Separator that will be used in output csv file.
+Do not use "," - it will be in conflict with JSON values
+
+'-b', '--bg_tasks' : Number of event loop tasks each process will create.
+If one runs huge amount of processes from a single machine this param could be decreased to reduces amount of used memory.
+Should not be less than 2.
+
+'-r', '--refresh' : Specifies the rate overall statistics is being updated in number of processed txns.
+If one runs huge amount of processes from a single machine this param could be decreased to reduces amount of used memory.
+Small values require lots of input/output.
+Should not be less than 10.
+
+'-k', '--kind' : Specifies the type of requests to be sent to pool.
+Supported txns:
+* nym - Create did
+* schema - Create schema
+* attrib - Create attribute
+* definition - Create credential definition
+* def_revoc - Create revocation registry
+* entry_revoc - Create revocation registry entry
+* get_nym - Get did
+* get_attrib - Get attribute
+* get_schema - Get schema
+* get_definition - Get credential definition
+* get_def_revoc - Get revocation registry
+* get_entry_revoc - Get revocation registry entry
+
+## Examples
+
+To be able to send txns of one type only with randomly generated data:
```

Comment:
Please use ordered list for each of the examples

Reply:
fixed